source/extensions/filters/http/grpc_json_transcoder/json_transcoder_filter.cc

```diff
@@ -141,10 +141,12 @@ ProtobufUtil::Status JsonTranscoderConfig::createTranscoder(
     const Http::HeaderMap& headers, ZeroCopyInputStream& request_input,
     google::grpc::transcoding::TranscoderInputStream& response_input,
     std::unique_ptr<Transcoder>& transcoder, const Protobuf::MethodDescriptor*& method_descriptor) {
+  if (Grpc::Common::hasGrpcContentType(headers)) {
+    return ProtobufUtil::Status(Code::INVALID_ARGUMENT,
+                                "This request headers has application/grpc content-type");
```

Comment:
@lizan I'm not sure what to put as status code and message here. Need your suggestion. Thank you!

Reply:
I think InvalidArgument is good for now, alternately it can just return a bool, since the filter doesn't check.

Reply:
nit: Remove "This " ("Request headers has...")

Reply:
@mattklein123 got it, removed in 99592ef.